### PR TITLE
Remove the old Window cython sources

### PR DESCRIPTION
**Pull request by yunline**

### CODE REVIEW

File: src_c/cython/pygame/_sdl2/video.pyx [hunk 1]
```diff
@@ -144,380 +144,8 @@ def messagebox(title, message,
...
-        """
-        if SDL_SetWindowModalFor(self._win, parent._win):
-            raise error()
-
-    def __dealloc__(self):
-        if self._is_borrowed:
-            return
-        self.destroy()
+globals()["Window"] = Window
+globals()["_Window"] = Window
```
Comment (Starbuck5) [inline src_c/cython/pygame/_sdl2/video.pyx:148]: Why? Window is already in the global namespace here, and _Window does not to be a preserved API.

**Review by Starbuck5 — APPROVED** [review]

File: buildconfig/stubs/pygame/_sdl2/window.pyi [hunk 2]
```diff
@@ -1 +1,3 @@
-from pygame._sdl2.video import _Window as Window
+from pygame._sdl2 import video
+
+Window: type[video.Window]
```
Comment (ankith26) [inline buildconfig/stubs/pygame/_sdl2/window.pyi:3]: Hmm this is a bit of a weird pattern, I'd suggest something like (untested, but should work)

```suggestion
from pygame._sdl2.video import Window as Window
```
This actually has another advantage apart from being concise, the current code would error on python < 3.9

Comment (yunline) [reply buildconfig/stubs/pygame/_sdl2/window.pyi]: I thought this solution causes linter error, but it doesn't. (

**Review by ankith26 — APPROVED** [review]
I'm not really familiar with `Window` and `_sdl2` but approving this nonetheless, thanks!

**Pull request merged**